### ISSUE TITLE
Sim deployed with partial translations

### ISSUE THREAD

**Issue by Denz1994** [phetsims/build-a-molecule#214]
It was noted by @oliver-phet that the recently published BAM has a set of translated sims that are partially translated (titles and phet menu). I followed the normal `grunt production` procedures as documented in https://github.com/phetsims/phet-info/blob/master/deployment-info/chipper-2.0.md but after consulting with @jonathanolson and @jbphet it seems to be that this sim pulled java strings early in its development. The first HTML dev version was in 2013 and as far as our most recent porting efforts, that work started in mid-2019.

**Comment by Denz1994** [comment]
@jbphet suggested a motion to remove the translated sims from the website and phet-server, then delete the translated strings in babel. This should give the sim a clean slate. It was also suggested to be careful about removing legit translations that may have been submitted since the last day since the sim went live.

**Comment by Denz1994** [comment]
@mattpen and I took a look at the translated sims and determined based on the creation of the translated sims on phet-server no translator submissions were submitted. All the translations were submitted at the same time right when I did the deploy. We went ahead deleted them from phet-server.

Because the en sim page and the translated sim pages are all cached at different levels, tomcat will be delayed in updating the page. What this means is that the partially translated sims will be present on the page for another estimated 24 hours until the cache is refreshed.
